modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -207,6 +164,14 @@ res = requests.post(
 pretty(res)
 ```
 
+--
+rpk::
++
+--
+
+```bash
+rpk registry schema create sensor-value ~/code/tmp/sensor_sample.avro
```

Comment:
@r-vasquez Can rpk accept the json/yaml data like with the curl and Python examples, or will it only take a path to a file?

Reply:
For now it only takes a path to a file

Reply:
The path to the Avro file is passed via a flag: `--schema`

```suggestion
rpk registry schema create sensor-value --schema ~/code/tmp/sensor_sample.avro
```